pkg/cmd/clientConfig.go

```diff
@@ -77,35 +73,15 @@ kubectl plugin mobile get clientconfig`,
 				return err
 			}
 
-			ms := listServices(ns, ccc.k8Client)
-			for _, svc := range ms {
-				var svcConfig *ServiceConfig
-				var err error
-				includedService := true
-				configMap, err := ccc.k8Client.CoreV1().ConfigMaps(ns).Get(svc.Name, v1.GetOptions{})
-				// ignoring not found as some services will not have this public configmap
+			filter := v1.ListOptions{LabelSelector: fmt.Sprintf("mobileClientId=%s", clientID)}
```

Comment:
this label should  be `clientId` the APBs will be updated to use this string as the key.